nexus/users/models.py

```diff
@@ -30,21 +32,49 @@ def create_superuser(self, email: str, password: str, **extra_fields):
 
 
 class User(AbstractBaseUser, UUIDModel, PermissionsMixin):
-    first_name = models.CharField(_('First Name'), max_length=120, blank=True)
-    last_name = models.CharField(_('Last Name'), max_length=120, blank=True)
+
+    GENDER_CHOICES = Choices(
+        ('NOT_SELECTED', 'NA', _('Not Selected')),
+        ('MALE', 'M', _('Male')),
+        ('FEMALE', 'F', _('Female')),
+        ('OTHERS', 'O', _('Others')),
+    )
+
+    TSHIRT_SIZE_CHOICES = Choices(
+        ('NOT_SELECTED', 'NA', _('Not Selected')),
+        ('SMALL', 'S', _('Small')),
+        ('MEDIUM', 'M', _('Medium')),
+        ('LARGE', 'L', _('Large')),
+        ('EXTRA_LARGE', 'XL', _('Extra Large')),
+        ('DOUBLE_EXTRA_LARGE', 'XXL', _('Double Extra Large')),
+    )
+
+    first_name = models.CharField(_('First Name'), null=False, blank=True, max_length=120)
+    last_name = models.CharField(_('Last Name'), null=False, blank=True, max_length=120)
     # https://docs.djangoproject.com/en/1.11/ref/contrib/postgres/fields/#citext-fields
-    email = CIEmailField(_('email address'), unique=True, db_index=True)
-    is_staff = models.BooleanField(_('staff status'), default=False,
+    email = CIEmailField(_('Email Address'), null=False, blank=False, unique=True, db_index=True)
```

Comment:
Putting `null=False` and `blank=False` is not required since the `UserManager` would not allow a user creating without `email` and `password`. Hence, it's redundant.

Reply:
Yes, that's true! But, in `coding_rules.md` is it written that:
``` All fields have explicit `blank` and `null` parameters. ```
So I thought it is required to show them explicitly.

Reply:
Yeah. But in this case, it is not required.